### PR TITLE
add medihover borg sprites

### PR DESCRIPTION
**Pull request by utahclock**
adds sprites for medihover borg, needs coder input to add to borg menu

### CODE REVIEW

**Comment by Notamaniac** [comment]
You've added a copy of the robot.dmi to the main yogstation folder. You also need to set the eye delay to 3 and 8; if you squint you'll see them bobbing up and down rapidly, way out of sync with the sprite

**Comment by utahclock** [comment]
ok added the code lines that allows menu selection for medihover. :)

**Comment by Notamaniac** [comment]
I think you need to also add a line under /mob/living/silicon/robot/update_icons() (same file for the mediborg selection, just use ctrl f to find it) for the eyes to actually work, again I'm not exactly byond fluent but I think you need to add something along the lines of

`if("medihover")
                overlays += "eyes-medihover"`

**Comment by utahclock** [comment]
there is no update_icons file under robots , so im lost as to where to add that line of code.

**Comment by Notamaniac** [comment]
Line 820 in code/modules/mob/living/silicon/robot/robot.dm (the one you edited so you could pick it). I didn't mean another file sorry.

**Comment by utahclock** [comment]
right eyes added now . thanks for help maniac

File: icons/mob/robot.dm [hunk 1]
```diff
@@ -0,0 +1,1246 @@
+/mob/living/silicon/robot
```
Comment (AsV9) [inline icons/mob/robot.dm]: wat is this file doing here

**Comment by utahclock** [comment]
Asv file was in wrong spot its fixed now.

**Pull request merged**